src/outlier_detection/auto_outliers.py

```diff
@@ -99,22 +99,6 @@ def flag_outliers(
     # Filter for valid sampled data and positive values in the value column
     filtered_df = filter_valid(df, value_col)
 
-    def _normal_round(x: float) -> int:
```

Comment:
I called a single leading underscore to signify that it's an internal use function. Yet, if you are going to unit test it, you will need to import it, so it is correct to remove the underscore.